grout/models.py

```diff
@@ -24,14 +27,34 @@ class Meta(object):
         abstract = True
 
 
-class SchemaModel(GroutModel):
+class RecordType(GroutModel):
+    """
+    Store metadata for a class of Records.
+    """
+    label = models.CharField(max_length=64)
+    plural_label = models.CharField(max_length=64)
+    description = models.TextField(blank=True, null=True)
+    active = models.BooleanField(default=True)
+
+    def get_current_schema(self):
+        schemas = self.schemas.order_by('-version')
+        return schemas[0] if len(schemas) > 0 else None
+
+
+class RecordSchema(GroutModel):
```

Comment:
Previously, `RecordSchema` had inherited from an abstract class `SchemaModel` that provided some JSON-validating methods. No other model was using `SchemaModel`, so I've merged it together with `RecordSchema` to simplify the data model a bit.